packages/wrangler/src/dev/miniflare.ts

```diff
@@ -837,6 +842,42 @@ export async function buildMiniflareOptions(
 		}
 	}
 
+	const assetServerModulePath = require.resolve(
+		"@cloudflare/workers-shared/dist/asset-server-worker.mjs"
+	);
+	const assetServerConfigPath = require.resolve(
+		"@cloudflare/workers-shared/asset-server-worker/wrangler.toml"
+	);
+	let assetServerConfig: Config | undefined;
+
+	try {
+		assetServerConfig = readConfig(assetServerConfigPath, {});
+	} catch (err) {
+		//  ignore and continue with some predefined default config
+	}
+
+	const assetServerWorker: WorkerOptions | undefined = config.experimentalAssets
+		? {
+				name: assetServerConfig?.name ?? "asset-server",
```

Comment:
dbl-check: are these sensible defaults?

Reply:
Again, I'd just be tempted to write a `requireConfig()` typeguard sort of function and blow up if we fail to validate that before giving to miniflare. What do you think?

Reply:
totes agree! I should have prefixed this question with, if we don't throw on config read... I'll remove

Reply:
✅